src/main/java/com/mraof/minestuck/entity/underling/GiclopsEntity.java

```diff
@@ -137,63 +87,52 @@ public void die(DamageSource cause)
-	{
-		AxisAlignedBB realBox = this.getBoundingBox();
-		this.setBoundingBox(new AxisAlignedBB(realBox.minX, realBox.maxY - 1, realBox.minZ, realBox.maxX, realBox.maxY, realBox.maxZ));
-		boolean result = super.updateFluidHeightAndDoFluidPushing(fluidTag, fluidFactor);
-		this.setBoundingBox(realBox);
-		return result;
-	}
-	
-	
-	@Override
-	public void move(MoverType typeIn, Vector3d pos)
-	{
-		AxisAlignedBB realBox = this.getBoundingBox();
-		double minX = pos.x > 0 ? realBox.maxX - pos.x : realBox.minX;
-		/*				y > 0 ? realBox.maxY - y : realBox.minY,*/
-		double minY = realBox.minY;
-		double minZ = pos.z > 0 ? realBox.maxZ - pos.z : realBox.minZ;
-		double maxX = pos.x < 0 ? realBox.minX - pos.x : realBox.maxX;
-		double maxY = pos.y < 0 ? realBox.minY - pos.y : realBox.maxY;
-		double maxZ = pos.z < 0 ? realBox.minZ - pos.z : realBox.maxZ;
-		this.setBoundingBox(new AxisAlignedBB(minX, minY, minZ, maxX, maxY, maxZ));
-		super.move(typeIn, pos);
-		AxisAlignedBB changedBox = this.getBoundingBox();
-		this.setBoundingBox(realBox.move(changedBox.minX - minX, changedBox.minY - minY, changedBox.minZ - minZ));
-		this.setLocationFromBoundingbox();
-	}
-	
-	@Override
-	public PartGroup getGroup()
-	{
-		return partGroup;
-	}
-	
-	/**
-	 * Will get destroyed next tick.
-	 */
-	@Override
-	public void remove()
-	{
-		super.remove();
-		partGroup.updatePositions();
-	}
+//	//Reduced lag is worth not taking damage for being inside a wall
+//	@Override
+//	public boolean isInWall()
+//	{
+//		return false;
+//	}
+//
+//	//Only pay attention to the top for water
+//
+//	@Override
+//	public boolean updateFluidHeightAndDoFluidPushing(ITag<Fluid> fluidTag, double fluidFactor)
+//	{
+//		AxisAlignedBB realBox = this.getBoundingBox();
+//		this.setBoundingBox(new AxisAlignedBB(realBox.minX, realBox.maxY - 1, realBox.minZ, realBox.maxX, realBox.maxY, realBox.maxZ));
+//		boolean result = super.updateFluidHeightAndDoFluidPushing(fluidTag, fluidFactor);
+//		this.setBoundingBox(realBox);
+//		return result;
+//	}
+	
+	
+//	@Override
+//	public void move(MoverType typeIn, Vector3d pos)
+//	{
+//		AxisAlignedBB realBox = this.getBoundingBox();
+//		double minX = pos.x > 0 ? realBox.maxX - pos.x : realBox.minX;
+//		/*				y > 0 ? realBox.maxY - y : realBox.minY,*/
+//		double minY = realBox.minY;
+//		double minZ = pos.z > 0 ? realBox.maxZ - pos.z : realBox.minZ;
+//		double maxX = pos.x < 0 ? realBox.minX - pos.x : realBox.maxX;
+//		double maxY = pos.y < 0 ? realBox.minY - pos.y : realBox.maxY;
+//		double maxZ = pos.z < 0 ? realBox.minZ - pos.z : realBox.maxZ;
+//		this.setBoundingBox(new AxisAlignedBB(minX, minY, minZ, maxX, maxY, maxZ));
+//		super.move(typeIn, pos);
+//		AxisAlignedBB changedBox = this.getBoundingBox();
+//		this.setBoundingBox(realBox.move(changedBox.minX - minX, changedBox.minY - minY, changedBox.minZ - minZ));
+//		this.setLocationFromBoundingbox();
+//	}
```

Comment:
I'd suggest using multiline comments (/* */) or outright removing the code when getting rid of multiple lines of code like this, as it makes these diffs easier to look through.

Reply:
I just ctrl + / 'd the code, i think the issue is related to the tab/space indent mixing